phpseclib/Crypt/DH.php

```diff
@@ -84,7 +85,7 @@ public static function createParameters(...$args): Parameters
         $params = new Parameters();
         if (count($args) == 2 && $args[0] instanceof BigInteger && $args[1] instanceof BigInteger) {
             //if (!$args[0]->isPrime()) {
-            //    throw new \InvalidArgumentException('The first parameter should be a prime number');
+            //    throw new \phpseclib3\Exception\InvalidArgumentException('The first parameter should be a prime number');
```

Comment:
Would prob be better to just remove the \ instead of adding the full namespacing.